syft/execution/communication.py

```diff
@@ -14,7 +14,9 @@
     "mid_get",
     "move",
     "remote_get",
-    "remote_send",
+    "remote_copy_to",
```

Comment:
I realize I suggested this name, but I might have a better one. What do you think about `remote_send_copy()`?

Reply:
I like remote copy to - not too opinionated